sierra_adapter/sierra_reader/src/main/scala/uk/ac/wellcome/platform/sierra_reader/modules/WindowManager.scala

```diff
@@ -61,11 +61,7 @@ class WindowManager @Inject()(
         val triedStatus = triedMaybeLastId
           .map {
             case Some(id) =>
-              // The Sierra IDs we store in S3 are prefixed with "b" or "i".
-              // Remove the first character
-              val unprefixedId = id.substring(1)
-
-              val newId = (unprefixedId.toInt + 1).toString
```

Comment:
Flagging this for extra review – this is the correct behaviour, but does this mean we’re missing Sierra data? In particular, I can’t see how this logic would ever cause us to fetch more than the first page of seven-digit Sierra IDs

(Why didn’t we put an `assert id.substring(0) in ('b', 'i')`, gah)

Reply:
You might have to explain this in person.

Reply:
When we first wrote the Sierra reader, the IDs we stored were of the form “b1234567”.

The Sierra API wants to see the “1234567” part, no “b” – so we discard it here.

Now IDs are stored in the form “1234567”, so this transformation would strip it to “234567”, and we’d request that from the Sierra API – which is wrong.

So I think we might be dropping all but the first page of results. 😕